app/users.php

```diff
@@ -1,87 +1,238 @@
-<?php require 'init.php';?>
+<?php require 'init.php'; ?>
+<?php include_once $_ENV['BASE_DIRECTORY'] . '/web-assets/app_nav/footer.php'; ?>
 <?php $page_title = 'Ghostmeme' ?>
 <?php
 
 
-    $error = NULL;
-    if(!isset($_SESSION['user']) && !isset($_COOKIE['user'])){
-        header('Location: ' . $_ENV['BASE_URL'] . '/login.php');
-    }
+echo $twig->render('layouts/user.layout.twig');
```

Comment:
This would be the last line of your php file, after you've set up all your variables.